### DISCUSSION TITLE
Icon Name images

### DISCUSSION THREAD

**Discussion by Senari7** [TobiasM95/WoW-Talent-Tree-Manager#25] [General]
It appears we can only access a limited selection of icons from a database specific to current Dragonflight talent/class trees. Is there a way to access more icons from perhaps the wowhead database? 

Thanks!

**Comment by TobiasM95** [comment]
You can import your own custom icons. 
`To include custom icons in TTM, put .png files with size 40x40 in the %%APPDATA%%(Roaming)/Wow Talent Tree Manager/resources/icons/custom/ directory (create it if it doesn't exist). Restart the application and you should see your custom icons being available. If they have the same name as currently existing icons, they will overwrite the original. You can use this to create your own versions of existing icons without changing original icons.`

**Comment by Senari7** [comment]
Thank you!
